### ISSUE TITLE
questions about label maps??

### ISSUE THREAD

**Issue by xiaohaipeng** [ZPdesu/SEAN#12]
pics in dir labels are of one channel ,the pixel values should be classification codes,but they are not consequent，like 0，1，2，3.... 。so when do one-hot label map，how does it work？？？

**Comment by ZPdesu** [comment]
- Training on 512*512 is very slow, so I didn't try it. If you wanna improve the training speed, the best way is combing the left-right labels (e.g. left eye and right eye).

- According to my experiments, IN only performs better during training, and the test performance would be worse than BN (e.g. FID score).

**Comment by xiaohaipeng** [comment]
Thank you for your reply.
I read your paper,but not very understand how you design your SEAN block . What is your consideration of  crossing style map features and segmentation mask features? Can we remove the segmentation mask branch ,only keep the style map branch?

**Comment by ZPdesu** [comment]
Yep, it still works, but the performance of quantitative evaluation will drop a bit
